src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_ui_language_change_notification.cc

```diff
@@ -59,17 +59,23 @@ OnUILanguageChangeNotification::~OnUILanguageChangeNotification() {}
 void OnUILanguageChangeNotification::Run() {
   LOG4CXX_AUTO_TRACE(logger_);
 
-  HMICapabilities& hmi_capabilities = hmi_capabilities_;
-
-  hmi_capabilities.set_active_ui_language(
+  hmi_capabilities_.set_active_ui_language(
       static_cast<hmi_apis::Common_Language::eType>(
           (*message_)[strings::msg_params][strings::language].asInt()));
 
+  std::vector<std::string> sections_to_update;
+  sections_to_update.push_back(hmi_response::language);
```

Comment:
@ydementieiev Same as above?

Reply:
@AByzhynar I changed it in all needed [place](https://github.com/LuxoftSDL/sdl_core/pull/35/commits/cdab57c9b89acd94155de176f4a223e55f72602a)